packages/react/src/components/SpinButton/SpinButton.base.tsx

```diff
@@ -176,14 +177,18 @@ export const SpinButtonBase: React.FunctionComponent<ISpinButtonProps> = React.f
     }
   }, [valueFromProps, previousValueFromProps, intermediateValue]);
 
-  const classNames = getClassNames(styles, {
-    theme: theme!,
-    disabled,
-    isFocused,
-    keyboardSpinDirection,
-    labelPosition,
-    className,
-  });
+  const classNames = getClassNames(
+    styles,
+    {
+      theme: theme!,
+      disabled,
+      isFocused,
+      keyboardSpinDirection,
+      labelPosition,
+      className,
+    },
+    shadowDom,
```

Comment:
Unfortunately it looks like we'll have to plumb this value through all components. Would love ideas to avoid this!